CorsixTH/Lua/entities/patient.lua

```diff
@@ -46,12 +46,15 @@ function Patient:Patient(...)
   self.vaccination_candidate = false
   -- Has the patient passed reception?
   self.has_passed_reception = false
+  self.go_home_reasons = {CURED = "cured", KICKED = "kicked", OVER_PRICED = "overpriced"}
 
   -- Is the patient trying to get to the toilet? ("yes", "no", or "no-toilets")
   self.going_to_toilet = "no"
 end
 
 function Patient:onClick(ui, button)
+  self.over_priced = true
```

Comment:
@sadger: Can we drop this variable now? It looks like it's only there for test/log purpose?